### PR TITLE
[TACHYON-1436] Fix listRecursive to work with hdfs

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1436

### CODE REVIEW

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: integration-tests/src/test/java/tachyon/underfs/UnderStorageSystemInterfaceIntegrationTest.java [hunk 1]
```diff
@@ -225,6 +225,8 @@ public void testList() throws IOException {
   @Test
   public void testListRecursive() throws IOException {
     String root = mUnderfsAddress;
+    // TODO(andrew): Should this directory be created in LocalTachyonCluster creation code?
```
Comment (calvinjia) [inline integration-tests/src/test/java/tachyon/underfs/UnderStorageSystemInterfaceIntegrationTest.java]: The ufs root should be available, does it not get created for hdfs profiles?

**Comment by calvinjia** [comment]
LGTM

**Comment by aaudiber** [comment]
Jenkins, test this please.

**Pull request merged**